Graphics/Renderers/CalamityRenderers/MiracleBlightRenderer.cs

```diff
@@ -68,7 +68,7 @@ public override void DrawToTarget(SpriteBatch spriteBatch)
             ActuallyDoPreDraw = true;
 
             // Draw every npc to a single target that should have the miracle blight visual.
-            foreach (NPC npc in Main.npc)
+            foreach (var npc in Main.npc)
```

Comment:
These 3 appear to have been done automatically by VS, as I did not manually make those changes. I'll ensure I double check each change to avoid anything slipping in in future. 👍